src/import-checker-code-action.ts

```diff
@@ -21,18 +33,15 @@ export class ImportCheckerCodeAction implements CodeActionProvider {
     return existsSync(`${workspace.rootPath}/node_modules/`);
   }
 
-  private extractPackageFromImport(text: string): string {
-    try {
-      const [,,packageName] = /^import (.*) from ['|"]+([^./]{0,})+['|"];$/.  exec(text) || ['', '', ''];
-      return packageName;
-    } catch (error) {
-      console.log(error);
-    }
+  private extractPackageFromImport(line: string): string {
```

Comment:
What's missing in this pattern? I prefer to use regex, it seems more clear for me.